addons/account_edi_peppol/models/res_partner.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api
+
+PEPPOL_COUNTRY_EAS = {
+    'HU': 9910,
+
```

Comment:
why such empty lines?

Reply:
For readability, the sequence is continuous, if peppol add new EAS or if we want to support other endpoint_scheme, it will be easier to add them here.